src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,252 @@
+		if ((Math.random()) < 0.001) {
+			losses += this.calculateWinnings(wager, turn - 1);
+			await message.author.settings.decrease(UserSettings.Money, losses);
+		}
+
+		await gameMessage.edit(new MessageEmbed()
+			.setColor(Slotmachine.FAIL_COLOUR)
+			.setTitle(message.language.tget('COMMAND_HIGHERLOWER_LOSE_TITLE'))
+			.setDescription(message.language.tget('COMMAND_HIGHERLOWER_LOSE_DESCRIPTION', number, losses))
+			.setFooter(message.language.tget('COMMAND_HIGHERLOWER_LOSE_FOOTER')));
+		return false;
+	}
+
+	private async end(game: HigherLowerGameData, message: KlasaMessage, cashout?: boolean) {
+		// Remove all the reactions and end the LLRC
+		await game.gameMessage.reactions.removeAll();
+		game.llrc.end();
+
+		// Should we need to cash out, proceed to doing that
+		if (cashout!) {
+			const { turn, wager } = game;
+			// Let the user know, and nullify the embed
+			await game.gameMessage.edit(message.language.tget('COMMAND_HIGHERLOWER_CASHOUT_INIT'), { embed: null });
+
+			// Calculate and deposit winnings for that game
+			const winnings = this.calculateWinnings(wager, turn - 1);
+			await message.author.settings.increase(UserSettings.Money, winnings);
+
+			// Let the user know we're done!
+			await game.gameMessage.edit(message.language.tget('COMMAND_HIGHERLOWER_CASHOUT', winnings));
+			return;
+		}
+
+		// Say bye!
+		await game.gameMessage.edit(new MessageEmbed()
+			.setColor(Slotmachine.FAIL_COLOUR)
+			.setTitle(message.language.tget('COMMAND_HIGHERLOWER_CANCEL_TITLE'))
+			.setDescription(message.language.tget('COMMAND_HIGHERLOWER_CANCEL_DESCRIPTION', message.author.username)));
+	}
+
+	private reactionInhibitor(message: KlasaMessage, gameMessage: KlasaMessage, reaction: LLRCData) {
+		// If there's no gameMessage, inhibit
+		if (!gameMessage) return true;
+
+		// If the message reacted is not the game's, inhibit
+		if (reaction.messageID !== gameMessage.id) return true;
+
+		// If the emoji reacted is not valid, inhibit
+		if (!Object.values(ReactionEmoji).includes(reaction.emoji.name as ReactionEmoji)) return true;
+
+		// If the user who reacted is the author, don't inhibit
+		if (reaction.userID === message.author.id) return false;
+
+		// Don't listen to herself
+		if (reaction.userID === this.client.user!.id) return true;
+	}
+
+	/**
+	 * @description Generates a random number between 0 and 100
+	 * @param previous The number we shouldn't get (usually the number we're comparing against
+	 */
+	private random(previous?: number) {
+		let number = previous!;
+		while (number === previous) {
+			number = (Math.random() * 100) | 0;
+		}
+		if (number > 100) number = 100;
+		return number;
+	}
+
+	private calculateWinnings(bet: number, attempts: number) {
+		if (attempts < 0) attempts = 1;
+		return Math.round(bet * (attempts <= 4 ? (Math.exp(attempts)) / 6 : (attempts * 5) - 10));
+	}
+
+}
+
+interface HigherLowerGameData {
+	llrc: LongLivingReactionCollector;
```

Comment:
```suggestion
	/** The game's reaction collector */
	llrc: LongLivingReactionCollector;
```

Lets add TSDoc because the abbreviation isn't directly obvious in the code

Reply:
(It actually is)